libgstc/libgstc_socket.c

```diff
@@ -72,7 +82,10 @@ gstc_socket_new (const char *address, const unsigned int port,
     goto out;
```

Comment:
Use different names for the pointer and the label. `out` is not a very descriptive name.

Reply:
The pointer's name has been changed to `socket` in the latest version, out remained as the label name since the is the standard in the rest of the code